### PR TITLE
VFB-224: Update local storage

### PR DESCRIPTION
**Pull request by rickaw**

### CODE REVIEW

File: supabase/tools/createStorage.ts [hunk 1]
```diff
@@ -23,21 +23,24 @@ async function createPublicBucket(supabase: SupabaseClient, name: string): Promi
...
-async function uploadCongestionZoneFile(supabase: SupabaseClient): Promise<void> {
+async function uploadCongestionZoneFile(
+    supabase: SupabaseClient,
+    bucketName: string
+): Promise<void> {
     const file = fs.readFileSync("supabase/tools/congestion-zone-postcodes.txt");
 
     const { error } = await supabase.storage
-        .from("public")
+        .from(bucketName)
```
Comment (8kristy) [inline supabase/tools/createStorage.ts:37]: Is it different for creating buckets and the "from" part needs to be the actual bucket name?

File: supabase/tools/createStorage.ts [hunk 2]
```diff
@@ -23,21 +23,24 @@ async function createPublicBucket(supabase: SupabaseClient, name: string): Promi
     });
 
     if (error) {
-        throw new Error(`Failed to create bucket called ${name}, error: ${error}`);
+        throw new Error(`Failed to create bucket called ${name}, error: ${error.message}`);
```
Comment (8kristy) [inline supabase/tools/createStorage.ts:26]: I assume this was just wrong before?

File: supabase/tools/createStorage.ts (hunk 1 above)
Comment (rickaw) [reply supabase/tools/createStorage.ts:37]: this is inconsistent with how we access the bucket in the edge function but this is in fact the syntax that works when i tested locally

Comment (rickaw) [reply supabase/tools/createStorage.ts:37]: which is a big shame

File: supabase/tools/createStorage.ts (hunk 2 above)
Comment (rickaw) [reply supabase/tools/createStorage.ts:26]: yup yup the error message came out as `[Object object]` so showing the message instead now

**Review by 8kristy — APPROVED** [review]

**Pull request merged**